scripts/mlp/end_effector/bezier_predef.py

```diff
@@ -204,7 +204,7 @@ def generateSmoothBezierTrajWithoutPredef(time_interval,placement_init,placement
     return ref_traj
 
 def generateSmoothBezierTraj(time_interval,placement_init,placement_end,numTry=None,q_t=None,phase_previous=None,phase=None,phase_next=None,fullBody=None,eeName=None,viewer=None):
-    if numTry > 0 :
+    if numTry and numTry > 0 :
```

Comment:
I understand this, but it feels a bit awkward. You can be a bit more explicit with `if numTry is not None and numTry > 0:`, but that's not mandatory.

Reply:
done.